scripts/build_non_firebase_sdks.sh

```diff
@@ -28,11 +28,11 @@ echo "[" >> "${ZIP_POD_JSON}"
 for sdk in "${NON_FIREBASE_SDKS[@]}"
 do
   if [ ${sdk} == "FirebaseFirestoreSwift" ]; then
-    echo "{\"name\":\"FirebaseFirestoreSwift\", \"version\" : \"> 8.2-beta\"}" >>  "${ZIP_POD_JSON}"
+    echo "{\"name\":\"FirebaseFirestoreSwift\", \"version\" : \"> 8.2\"}" >>  "${ZIP_POD_JSON}"
   elif [ ${sdk} == "FirebaseStorageSwift" ]; then
-    echo "{\"name\":\"FirebaseStorageSwift\", \"version\" : \"> 8.12-beta\"}" >>  "${ZIP_POD_JSON}"
+    echo "{\"name\":\"FirebaseStorageSwift\", \"version\" : \"> 8.12\"}" >>  "${ZIP_POD_JSON}"
   elif [ ${sdk} == "FirebaseRemoteConfigSwift" ]; then
-    echo "{\"name\":\"FirebaseRemoteConfigSwift\", \"version\" : \"> 8.12-beta\"}" >>  "${ZIP_POD_JSON}"
+    echo "{\"name\":\"FirebaseRemoteConfigSwift\", \"version\" : \"> 8.12\"}" >>  "${ZIP_POD_JSON}"
```

Comment:
cc @paulb777 are these needed anymore with the zip builder building the Swift pods?

Reply:
Yes, these should all be deleted. The ci scripts should have already been updated not to request them.

Reply:
@ryanwilson @paulb777 , so this chunk should be removed?

https://github.com/firebase/firebase-ios-sdk/blob/fcd651bf35ec4a96f1ca743f496e33c82d321e43/scripts/build_non_firebase_sdks.sh#L30-L38

Reply:
I believe line 37 should remain.

Reply:
Ahh i see now. Thanks!